src/components/Courses/Courses.js

```diff
@@ -0,0 +1,156 @@
+import React, { useState, useEffect } from "react";
+import axios from 'axios';
+
+import classes from "./Courses.module.css";
+
+const Courses = (props) => {
+
+  // Initial state for tracking catalogs
+  const [coursesState, setCoursesState] = useState({
+    courses: null,
+    isLoading: false,
+    error: null
+  });
+
+  // TODO: Replace this DAU call with the selected in catalog name
+  const catalog_courses_api_url = process.env
+    .REACT_APP_XIS_COMPOSITELEDGER_API + "?provider=DAU";
```

Comment:
Will this TODO be addressed prior to merging the code?  If not, we should log a ticket so we don't forget about it.

Reply:
https://jira.deloitte.com/browse/ECC-634
We will need this until we are done with the XMS Cleanup